lib/matplotlib/tests/test_text.py

```diff
@@ -714,3 +714,10 @@ def test_update_mutate_input():
 def test_invalid_color():
     with pytest.raises(ValueError):
         plt.figtext(.5, .5, "foo", c="foobar")
+
+
+@image_comparison(['text_pdf_kerning.pdf'])
+def test_pdf_kerning():
+    plt.figure()
+    ax = plt.subplot(1, 1, 1)
+    ax.text(0.1, 0.5, "ATATATATATATATATATA", size=30)
```

Comment:
please use `fig.text()` (or `plt.figtext()`) instead, which avoids building an Axes and thus results in smaller baseline files.

Reply:
Sure :+1: